templates/azure-functions/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "build": "remix build",
+    "dev": "swa start ./public --run \"remix dev\" --api-location .",
+    "postinstall": "func init --worker-runtime node --language typescript"
+  },
+  "dependencies": {
+    "@remix-run/azure-functions": "*",
+    "@remix-run/react": "*",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@azure/functions": "^3.0.0",
+    "@azure/static-web-apps-cli": "^0.8.3",
+    "@remix-run/dev": "*",
+    "@remix-run/eslint-config": "*",
+    "@remix-run/serve": "*",
```

Comment:
Not needed I think, as we already have `@azure/functions`?

```suggestion
```

Reply:
The package `@azure/functions` isn't a hosting package, it's just a types package.

Reply:
Then this one should go into `dependencies` I think?